docs/api/v1.47.yaml

```diff
@@ -9251,6 +9251,19 @@ paths:
             all tags of the given image that are present in the local image store
             are pushed.
           type: "string"
+        - name: "platform"
+          type: "string"
+          in: "query"
+          description: |
+            JSON encoded OCI Platform to select a platform-specific manifest
+            matching this platform to be pushed.
+            If not provided, full multi-platform image will be pushed.
+            Note: Selection is only supported with the containerd image store.
+              With graphdrivers image store, it will only check that the actual
+              platform of the image matches the provided platform and error out
+              otherwise.
+
+            Example: {"os": "linux", "architecture": "arm", "variant": "v5"}
```

Comment:
Looks like this still needs to be updated

Reply:
Oh; perhaps should be in backticks as well (haven't checked if that looks good in the generated docs though, but I think we do so in other places)